foundationdb/src/tuple/mod.rs

```diff
@@ -35,6 +36,25 @@ pub enum Error {
     FromUtf8Error(FromUtf8Error),
 }
 
+/// Tracks the depth of a Tuple decoding chain
+#[derive(Copy, Clone)]
+pub struct TupleDepth(usize);
+
+impl TupleDepth {
+    fn new() -> Self {
+        TupleDepth(0)
+    }
+
+    fn increment(&self) -> Self {
```

Comment:
I don't know what happened to my earlier comment but doesn't `increment` need to be public? Won't someone implementing the traits externally need access to it?

Reply:
Yes. You are correct. I will edit.